documentation/domains/Domain.md

```diff
@@ -89,6 +90,15 @@ The current status of the operation of the WebLogic domain. Updated automaticall
 | `serverStartPolicy` | string | The strategy for deciding whether to start a WebLogic Server instance. Legal values are NEVER, or IF_NEEDED. Defaults to IF_NEEDED. More info: https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#starting-and-stopping-servers. |
 | `serverStartState` | string | The WebLogic runtime state in which the server is to be started. Use ADMIN if the server should start in the admin state. Defaults to RUNNING. |
 
+### Common Mount Volume
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `medium` | string | The emptyDir volume medium. This is an advanced setting that rarely needs to be configured. Defaults to unset, which means the volume's files are stored on the local node's file system for the life of the pod. |
+| `mountPath` | string | The common mount path. The files in the path are populated from the same named directory in the images supplied by each container in 'commonMount.containers'. Defaults to '/common'. |
```

Comment:
1) commonMount containers --> 'serverPod.commonMount'
2) Add sentence:  "Each common mount volume must be configured with a different common mount path."